src/main/features/response/views/reject-all-of-claim.njk

```diff
@@ -13,15 +13,16 @@
       <form method="post" class="analytics-submit-event-trigger" data-event-action="Response: Select full admission option" data-event-label-from="option" novalidate>
         {{ csrfProtection(csrf) }}
 
-        <legend class="visually-hidden">{{ t('I reject all of the claim') }}</legend>
+        <legend class="visually-hidden">{{ t('Why do you reject the claim?') }}</legend>
```

Comment:
same here about the legend being duplicated?

Reply:
Good spot, we missed that at some point.

Reply:
we noticed similar things in DAC, so on the look out for it now